packages/server/lib/scaffold.js

```diff
@@ -157,7 +157,7 @@ module.exports = {
     return this.verifyScaffolding(folder, () => {
       debug(`copying example.json into ${folder}`)
 
-      return this._copy('fixtures/example.json', folder, config)
```

Comment:
I`d like to ask @cypress-io/test-runner team to review these changes. I moved the scaffolding files out from the server to `@packages/example` in order to reuse with the new client wizard. 

I triple-check that it works, tests were not updated but I still want to be sure I didn't break something. Thanks 😉